addons/edi/models/edi_document.py

```diff
@@ -260,6 +266,12 @@ class EdiDocument(models.Model):
 
     fail_fast = fields.Boolean(related="doc_type_id.fail_fast", readonly=True)
 
+    # Processing timing statistics
+    doc_stat_ids = fields.One2many(
+        "edi.document.stats", "doc_id", string="Document Statistics", readonly=True, index=True
```

Comment:
nit: `ondelete="cascade"` to prevent any leftover `edi.document.stats` records if an EDI is deleted

Reply:
Or it might be the other way round, in which case - you've already done it :laughing: